### PR TITLE
Add angular autofields

### PR DESCRIPTION
**Pull request by JustMaier**
https://github.com/JustMaier/angular-autoFields-bootstrap

### CODE REVIEW

File: ajax/libs/angular-autofields/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "data-binding",
+        "generate"
+    ],
+    "license": "MIT",
+    "homepage": "https://github.com/justmaier/angular-autoFields-bootstrap",
+    "author": {
+        "name": "Justin Maier",
+        "email": "just.maier@gmail.com"
+    },
+    "npmName": "angular-autofields-bootstrap",
```
Comment (PeterDaveHello) [inline ajax/libs/angular-autofields/package.json]: npm ERR! 404 404 Not Found: angular-autofields-bootstrap
npm ERR! 404 
npm ERR! 404 'angular-autofields-bootstrap' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

**Comment by JustMaier** [comment]
strange: https://www.npmjs.com/package/angular-autoFields-bootstrap

**Comment by PeterDaveHello** [comment]
It's case sensitive, so you'll need to update that field, thanks.

**Comment by PeterDaveHello** [comment]
And you only committed minified files, but in the auto-update setting including the all js files, I think they should be the same structure.

**Comment by JustMaier** [comment]
Made the adjustments you mentioned.

**Comment by JustMaier** [comment]
Thanks!

**Pull request merged**